Gemfile

```diff
@@ -1,10 +1,10 @@
 source 'https://rubygems.org'
 
-ruby '2.4.0'
+ruby '2.5.1'
 
 gem 'puma'
-gem 'rails', '~> 4.2'
-gem 'pg', '0.21.0'
+gem 'rails', '~> 5.0.0'
+gem 'pg'
```

Comment:
The `1.x` version of the PG gem does not work with Rails.

Reply:
@Drenmi thanks, fixed now. Didn't pick it up since it seems the app doesn't actually use a DB? Is that planned for the future?

Reply:
Supposedly, Rails 5.1.5 onwards is compatible with `pg-1.0.0` https://github.com/rails/rails/issues/31673#issuecomment-372099518

This fix could come free with an upgrade to Rails 5.1

However we still need to evaluate https://github.com/rubysg/rubysg-reboot/issues/261